packages/vscode/src/extension.ts

```diff
@@ -84,6 +84,26 @@ export async function activate(context: ExtensionContext): Promise<void> {
     },
   }
 
+  // enable fileWatcher to watch .prisma folder inside node_modules
+  const value: string = JSON.stringify(
+    workspace.getConfiguration().get('files.watcherExclude'),
+  )
+  if (value.includes('**/node_modules/*/**')) {
+    const replacedValue = value.replace(
+      '**/node_modules/*/**',
+      '**/node_modules/{[^.],?[^p],??[^r],???[^i],????[^s],?????[^m]}*',
```

Comment:
this is weird

Reply:
it's a way to exclude all files except a specific folder as described here: https://github.com/microsoft/vscode/issues/869#issuecomment-630086813

Reply:
I've added a comment there as well.